resource/packages/apt/preparer.go

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2016 Asteris, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apt
+
+import (
+	"github.com/asteris-llc/converge/load/registry"
+	"github.com/asteris-llc/converge/resource"
+)
+
+// Preparer for Content
+//
+// Content renders content to disk
+type Preparer struct {
+	// Content is the file content. This will be rendered as a template.
+	Name string `hcl:"name"`
+
+	// Destination is the location on disk where the content will be rendered.
+	State string `hcl:"state"`
```

Comment:
If either of these parameters are required, you should add the `required:"true"` tag. You should also include a `valid_values:"present,absent"` tag for State